### PR TITLE
[Cache] Adding Tiered Cache content

### PR DESCRIPTION
**Pull request by dcpena**
Adding content for Tiered Cache for all as part of [PCX-2163](https://jira.cfops.it/browse/PCX-2163).

### CODE REVIEW

File: products/cache/src/content/about/tiered-cache.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Tiered Cache
+pcx-content-type: concept
+---
+
+# Tiered Cache
+
+Tiered Cache uses the size of Cloudflare's network to reduce requests to customer origins by dramatically increasing cache hit ratios. With data centers around the world, Cloudflare caches content very close to end users, but if a piece of content is not in cache, the Cloudflare edge data centers must contact the origin server to receive the cacheable content.
```
Comment (marciocloudflare) [inline products/cache/src/content/about/tiered-cache.md]:
```suggestion
Tiered Cache uses the size of Cloudflare's network to reduce requests to customer origins by dramatically increasing cache hit ratios. With data centers around the world, Cloudflare caches content very close to end users. However, if a piece of content is not in cache, the Cloudflare edge data centers must contact the origin server to receive the cacheable content. 
```

Comment (marciocloudflare) [reply products/cache/src/content/about/tiered-cache.md]: To break a big sentence into two. Wdyt?

File: products/cache/src/content/how-to/enable-tiered-cache.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Enable Tiered Cache
+pcx-content-type: how-to
+---
+
+# Enable Tiered Cache
+
+1. Log into your [Cloudflare dashboard](https://dash.cloudflare.com).
```
Comment (marciocloudflare) [inline products/cache/src/content/how-to/enable-tiered-cache.md]: This is missing something after the log in. In the Steps / Tasks / Procedures part of our content strategy, the example says that we should have `Log in to the Cloudflare dashboard and select your account. Go to Web Analytics.` OR `After logging in to the Cloudflare dashboard, select your domain and go to Traffic.`

File: products/cache/src/content/how-to/enable-tiered-cache.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+1. Click **Caching** > **Tiered Cache**.
+1. From **Argo Tiered Cache**, toggle the button to enabled.
```
Comment (marciocloudflare) [inline products/cache/src/content/how-to/enable-tiered-cache.md]:
```suggestion
1. From **Argo Tiered Cache**, toggle the button to **enabled**. 
```

File: products/cache/src/content/how-to/enable-tiered-cache.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Log into your [Cloudflare dashboard](https://dash.cloudflare.com).
+1. Click **Caching** > **Tiered Cache**.
+1. From **Argo Tiered Cache**, toggle the button to enabled. 
+
+After enabling Tiered Cache, you are automatically enrolled in Smart Tiered Cache. 
+
+## Smart Tiered Cache 
+
+Smart Tiered Cache dynamically selects the single best upper tier for each of your website’s origins with no configuration required. Cloudflare dynamically finds the single best upper tier for an origin by using Cloudflare’s performance and routing data. Cloudflare collects latency data for each request to an origin, and uses the latency data to Using determine how well any upper-tier data center is connected with an origin. As a result, Cloudflare can select the best data center with the lowest latency to be the upper-tier for an origin.
```
Comment (marciocloudflare) [inline products/cache/src/content/how-to/enable-tiered-cache.md]:
```suggestion
Smart Tiered Cache dynamically selects the single best upper tier for each of your website’s origins with no configuration required, using our in-house performance and routing data. Cloudflare collects latency data for each request to an origin, and uses the latency data to determine how well any upper-tier data center is connected with an origin. As a result, Cloudflare can select the best data center with the lowest latency to be the upper-tier for an origin.
```

Comment (marciocloudflare) [reply products/cache/src/content/how-to/enable-tiered-cache.md]: Wdyt? The second sentence repeated a few things and that `Using` seemed out of place

**Review by marciocloudflare — APPROVED** [review]

File: products/cache/src/content/how-to/enable-tiered-cache.md (hunk 2 above)
Comment (dcpena) [reply products/cache/src/content/how-to/enable-tiered-cache.md]: Fixed! Thank you :)

**Pull request merged**